frontend/src/components/Events/Event.js

```diff
@@ -1,23 +1,22 @@
-import { Link } from "react-router-dom";
-import { Container, Typography, Button } from "@material-ui/core";
+import {useHistory} from "react-router-dom";
+import { Typography } from "@material-ui/core";
 import { styled } from "@material-ui/styles";
+import {ItemContainer} from "../Groups/Group";
 
 export const Typography1 = styled(Typography)({
   padding: "0.25rem",
 });
 
-export const Button2 = styled(Button)({
-  background: "#d3d3d3",
-  margin: "0.5rem",
-});
-
-export const SCLink = styled(Link)({
-  textDecoration: "none",
-});
 function Event(props) {
+  const history = useHistory();
   const startDate = new Date(props.info.startTime).toUTCString();
```

Comment:
Do you wanna destructure props? In big cases you can save many words! But most importantly you can see right at the top which props are passed into this component instead of checking parent component, the JSX, and mapState or mapDispatch to props.